### PR TITLE
Add missing init

### PR DESCRIPTION
**Pull request by bugadani**

### CODE REVIEW

File: template/src/bin/main.rs [hunk 1]
```diff
@@ -44,9 +50,7 @@ fn main() -> ! {
     .unwrap();
     //ENDIF
 
-    let delay = Delay::new();
```
Comment (bugadani) [inline template/src/bin/main.rs:47]: Um, what deleted this 🤔

**Comment by bjoernQ** [comment]
we probably shouldn't do it twice (in the BLE/WIFI case)

**Review by bjoernQ — APPROVED** [review]
Thanks

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 **/target
 src/template_files.rs
 template/Cargo.lock
+/.vscode
```
Comment (bjoernQ) [inline .gitignore:4]: but maybe not in template

Comment (bugadani) [reply .gitignore:4]: I specifically checked that it does not ignore the template

Comment (bugadani) [reply .gitignore:4]: image

**Comment by MabezDev** [comment]
Could we cut a release now? Not calling `init` is kinda bad :sweat_smile:

**Pull request merged**